src/contexts/AnalyticsContext.tsx

```diff
@@ -74,34 +75,27 @@ const AnalyticsSetup = ({ children }: { children: JSX.Element }) => {
 
   useEffect(() => {
     if (canCaptureUserProperties) captureUserProperties()
-  })
+  }, [canCaptureUserProperties, captureUserProperties])
 
   return children
 }
 
-const AnalyticsProvider = ({ children }: { children: JSX.Element }) => {
-  const analytics = useAppSelector((s) => s.settings.analytics)
-  const analyticsId = useAppSelector((s) => s.settings.analyticsId)
-  const settingsLoadedFromStorage = useAppSelector((s) => s.settings.loadedFromStorage)
-
-  return (
-    <PostHogProvider
-      apiKey={PUBLIC_POSTHOG_KEY}
-      options={{
-        host: PUBLIC_POSTHOG_HOST,
-        disableGeoip: true,
-        enable: settingsLoadedFromStorage && analytics && !!analyticsId,
```

Comment:
It could be that this line was the issue? :/

Reply:
The docs say:

```
    // If set to false, tracking will be disabled until `optIn` is called
```

https://posthog.com/docs/libraries/react-native#configuration-options